Gemfile

```diff
@@ -57,6 +57,8 @@ gem 'high_voltage', '~> 3.1'
 gem 'google_drive', require: false
 gem 'google-api-client'
 gem 'browser'
+gem 'notifications-ruby-client'
```

Comment:
We probably don't need this gem to be specified as it's a dependency of `mail-notify`